qp_klp/tests/data/sample-sheets/metagenomic/tellseq/good_sheet_absq_draft1.csv

```diff
@@ -0,0 +1,124 @@
+LS_8_7_2012_Rep1,LS.8.7.2012.Rep1,Plate_1,K2,LS_Timeseries_TellSeq_10283,C522,1,0.374865,1.49946,168.8581081,Plate_1.LS.8.7.2012.Rep1.K2
+LS_9_24_2013_Rep1_SS,LS.9.24.2013.Rep1.SS,Plate_1,J5,LS_Timeseries_TellSeq_10283,C577,1,0.12125,0.485,54.61711712,Plate_1.LS.9.24.2013.Rep1.SS.J5
+LS_9_24_2013_Rep2,LS.9.24.2013.Rep2,Plate_1,H1,LS_Timeseries_TellSeq_10283,C512,1,0.37503875,1.500155,168.9363739,Plate_1.LS.9.24.2013.Rep2.H1
+LS_9_28_2014_Rep1_SRE,LS.9.28.2014.Rep1.SRE,LS_Donor_SS_Samples_P1,E1,TELLSeq_March_MetaG_Run_55555,C503,1,0.37462688,1.4985075,168.7508446,LS_Donor_SS_Samples_P1.LS.9.28.2014.Rep1.SRE.E1
+LS_9_28_2014_Rep2,LS.9.28.2014.Rep2,Plate_1,L2,LS_Timeseries_TellSeq_10283,C530,1,0.376875,1.5075,169.7635135,Plate_1.LS.9.28.2014.Rep2.L2
+LS_9_28_2014_Rep2_SS,LS.9.28.2014.Rep2.SS,Plate_1,L5,LS_Timeseries_TellSeq_10283,C578,1,0.375081,1.500324,168.9554054,Plate_1.LS.9.28.2014.Rep2.SS.L5
+LS_9_3_2013_Rep1_SRE,LS.9.3.2013.Rep1.SRE,LS_Donor_SS_Samples_P1,K1,TELLSeq_March_MetaG_Run_55555,C506,1,0.374995,1.49998,168.9166667,LS_Donor_SS_Samples_P1.LS.9.3.2013.Rep1.SRE.K1
+LS_9_3_2013_Rep2,LS.9.3.2013.Rep2,Plate_1,K1,LS_Timeseries_TellSeq_10283,C506,1,0.37503,1.50012,168.9324324,Plate_1.LS.9.3.2013.Rep2.K1
+T_LS_7_12_15A,T.LS.7.12.15A,UROBIOME_TEST_MF_SAMPLES_P2,B3,TELLSeq_March_MetaG_Run_55555,C541,1,0.37512125,1.500485,168.973536,UROBIOME_TEST_MF_SAMPLES_P2.T.LS.7.12.15A.B3
+T_LS_7_15_15B,T.LS.7.15.15B,UROBIOME_TEST_MF_SAMPLES_P2,M3,TELLSeq_March_MetaG_Run_55555,C539,1,0.375022,1.500088,168.9288288,UROBIOME_TEST_MF_SAMPLES_P2.T.LS.7.15.15B.M3
+T_LS_7_15_15B_SRE,T.LS.7.15.15B.SRE,LS_Donor_SS_Samples_P1,B2,TELLSeq_March_MetaG_Run_55555,C525,1,0.375028125,1.5001125,168.9315878,LS_Donor_SS_Samples_P1.T.LS.7.15.15B.SRE.B2
+T_LS_7_19_15A,T.LS.7.19.15A,LS_Time_Series_ABSQ_P4,K3,TELLSeq_March_MetaG_Run_55555,C538,1,0.28075,1.123,126.463964,LS_Time_Series_ABSQ_P4.T.LS.7.19.15A.K3
+BLANK.O3,T.LS.7.19.15B,LS_Time_Series_ABSQ_P4,O3,TELLSeq_March_MetaG_Run_55555,C540,1,0.374806,1.499224,168.8315315,LS_Time_Series_ABSQ_P4.BLANK.O3
+T_LS_7_8_15A,T.LS.7.8.15A,LS_Time_Series_ABSQ_P4,D3,TELLSeq_March_MetaG_Run_55555,C542,1,0.3750915,1.500366,168.9601351,LS_Time_Series_ABSQ_P4.T.LS.7.8.15A.D3
+,,,,,,,,,,
+[Bioinformatics],,,,,,,,,,
+Sample_Project,QiitaID,BarcodesAreRC,ForwardAdapter,ReverseAdapter,HumanFiltering,library_construction_protocol,experiment_design_description,contains_replicates,,
+TELLSeq_March_MetaG_Run_55555,55555,TRUE,GATCGGAAGAGCACACGTCTGAACTCCAGTCAC,GATCGGAAGAGCGTCGTGTAGGGAAAGGAGTGT,TRUE,Knight Lab Kapa HyperPlus,tellseq metagenomics,FALSE,,
+LS_Timeseries_TellSeq_10283,10283,TRUE,GATCGGAAGAGCACACGTCTGAACTCCAGTCAC,GATCGGAAGAGCGTCGTGTAGGGAAAGGAGTGT,TRUE,Knight Lab Kapa HyperPlus,tellseq metagenomics,FALSE,,
+,,,,,,,,,,
+[Contact],,,,,,,,,,
+Sample_Project,Email,,,,,,,,,
+TELLSeq_March_MetaG_Run_55555,cbrenchy@gmail.com,,,,,,,,,

```

Comment:
There is a lot of information in here which I'm not sure is appropriate for a public repository

Reply:
Gotcha, I'll update it.